redis-sink/src/main/java/org/springframework/cloud/stream/module/redis/sink/RedisSinkApplication.java

```diff
@@ -0,0 +1,15 @@
+package org.springframework.cloud.stream.module.redis.sink;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.ComponentScan;
+
+@SpringBootApplication
+@ComponentScan(basePackageClasses= RedisSink.class)
```

Comment:
Can remove `@ComponentScan` as the default package that will be used from `@ComponentScan` in `SpringBootApplication` already contains RedisSink